fdbserver/worker.actor.cpp

```diff
@@ -900,8 +927,10 @@ ACTOR Future<Void> workerServer( Reference<ClusterConnectionFile> connFile, Refe
 							included = fileExists(d.filename + "1.fdq");
 						}
 						if(d.storedComponent == DiskStore::COMPONENT::TLogData && included) {
-							std::string basename = fileLogQueuePrefix.toString() + d.filename.substr(fileLogDataPrefix.size());
-							included = fileExists(basename + "0.fdq") && fileExists(basename + "1.fdq");
+							std::string logDataBasename = fileLogQueuePrefix.toString() + d.filename.substr(fileLogDataPrefix.size());
```

Comment:
change this to scan all files, and determine if enough of each component (disk queue and metadata store) exist